src/Mono.Android/Test/Resources/Resource.designer.cs

```diff
@@ -81,6 +81,9 @@ public partial class Drawable
 			// aapt resource value: 0x7f020004
 			public const int Icon = 2130837508;
 			
+			// aapt resource value: 0x7f020005
+			public const int image = 2130837509;
```

Comment:
Why is this `image` when the `@(AndroidResource)` is using `Image.9.png`? I'd expect this member to be `Image` (capital `I`).